src/bin/multi_lat.rs

```diff
@@ -95,8 +95,7 @@ fn multi_latency(rank: i32, pairs: i32, world: SystemCommunicator) {
     if world.rank() == 0 {
       let t: Vec<f64> = a.unwrap();
       let avg_latency: f64 = t.iter().sum::<f64>() / (pairs * 2) as f64;
-      // FIXME: need padding from format!
-      println!("{}           {} ", size, avg_latency);
+      println!("{:<28} {1:.2}", size, avg_latency);
```

Comment:
Doesn't pad in exactly the same way, but should be good enough.
I couldn't get the latency to align on its last digit instead of the first digit.